### PR TITLE
SpriteBatch small comment fixes

### PR DESCRIPTION
**Pull request by Shqrdx**
Just a small comments improvements.

### CODE REVIEW

File: MonoGame.Framework/Graphics/SpriteBatch.cs [hunk 1]
```diff
@@ -168,16 +171,17 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
         /// Submit a sprite for drawing in the current batch.
         /// </summary>
         /// <param name="texture">A texture.</param>
-        /// <param name="position">An optional drawing location on screen.</param>
-        /// <param name="destinationRectangle">An optional drawing bound.</param>
+        /// <param name="position">An optional drawing location on screen. If null - <param name="destinationRectangle"> using intended.</param></param>
```
Comment (tomspilman) [inline MonoGame.Framework/Graphics/SpriteBatch.cs]: I would say...

> The drawing location on screen or null if <param name="destinationRectangle"> is used.

File: MonoGame.Framework/Graphics/SpriteBatch.cs [hunk 2]
```diff
@@ -168,16 +171,17 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
...
+        /// <param name="destinationRectangle">An optional drawing bound. If null - <param name="position"> using intended.</param></param>
```
Comment (tomspilman) [inline MonoGame.Framework/Graphics/SpriteBatch.cs]: I would say...

```
The drawing bounds on screen or null if <param name="position"> is used.
```

File: MonoGame.Framework/Graphics/SpriteBatch.cs [hunk 3]
```diff
@@ -168,16 +171,17 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
         /// Submit a sprite for drawing in the current batch.
         /// </summary>
         /// <param name="texture">A texture.</param>
-        /// <param name="position">An optional drawing location on screen.</param>
-        /// <param name="destinationRectangle">An optional drawing bound.</param>
+        /// <param name="position">The drawing location on screen or null if <param name="destinationRectangle"> is used.</param></param>
```
Comment (tomspilman) [inline MonoGame.Framework/Graphics/SpriteBatch.cs]: I think this should use a `<paramref>` to reference the destinationRectangle parameter.

File: MonoGame.Framework/Graphics/SpriteBatch.cs [hunk 4]
```diff
@@ -168,16 +171,17 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
...
-        /// <param name="layerDepth">An optional depth of the layer of this sprite.</param>
+        /// <param name="origin">An optional center of rotation. Uses <see cref="Vector2.Zero"/> if null.</param>
+        /// <param name="rotation">An optional rotation of this sprite. 0 by default.</param>
+        /// <param name="scale">An optional scale vector. Uses <see cref="Vector2.One"/> if null.</param>
+        /// <param name="color">An optional color mask. Uses <see cref="Color.White"/> if null.</param>
+        /// <param name="effects">The optional drawing modificators. <see cref="SpriteEffects.None"/> by default.</param>
+        /// <param name="layerDepth">An optional depth of the layer of this sprite. 0 by default.</param>
         /// <exception cref="InvalidOperationException">Throwns if both <paramref name="position"/> and <paramref name="destinationRectangle"/> been used.</exception>
+        /// <remarks>This overload uses optional parameters.</remarks>
         /// <remarks>This overload requires only one of <paramref name="position"/> and <paramref name="destinationRectangle"/> been used.</remarks>
```
Comment (tomspilman) [inline MonoGame.Framework/Graphics/SpriteBatch.cs]: Note i'm not sure if having two `<remarks>` blocks is valid doc format or how it should behave.  I know currently SharpDoc is ignoring the second remark block.  I can fix that, but we need to be sure what we're doing is valid.

**Comment by Shqrdx** [comment]
@tomspilman I think its fine for merging. What do you think ?

**Comment by tomspilman** [comment]
Yeah...  this works.

**Pull request merged**